server/akka-http-server/src/main/scala/sttp/tapir/server/akkahttp/AkkaHttpServerInterpreter.scala

```diff
@@ -44,6 +48,28 @@ trait AkkaHttpServerInterpreter {
   ): Route = {
     new EndpointToAkkaServer(serverOptions).toRoute(serverEndpoints)
   }
+
+  def serialiseSSEToBytes: Source[ServerSentEvent, Any] => AkkaStreams.BinaryStream = sseStream =>
```

Comment:
I don't think this belongs to the interpreter. We have `AkkaWebSockets`, so maybe similarly, we could have a `AkkaServerSentEvents` object?

Reply:
Moved to `AkkaServerSentEvents` :)